var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -56,6 +58,7 @@ class Hpx(CMakePackage, CudaPackage):
 
     # Recommended dependency versions for 1.2.X
     depends_on('cmake@3.9.0:', when='@:1.2.1', type='build')
+    depends_on('boost@1.7w.0: cxxstd=17', when='@:stable')
```

Comment:
I think there's a typo in the version of Boost. Also is the version range `@:stable` intentional or do you mean `@stable`?